### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via codespell.

"Display source diff" to view the SVG source code that has been fixed. 

Edit: changes a .ui file so either postpone till post1.0 or cherrypick.

### CODE REVIEW

**Comment by maxwxyz** [comment]
@luzpaz could you please resolve the conflicts? TIA

File: src/Gui/Icons/Std_DemoMode.svg [hunk 1]
```diff
@@ -260,7 +260,7 @@
             <rdf:li>disc</rdf:li>
           </rdf:Bag>
         </dc:subject>
-        <dc:description>Shapes spining on a disc</dc:description>
+        <dc:description>Shapes spiNning on a disc</dc:description>
```
Comment (adrianinsaval) [inline src/Gui/Icons/Std_DemoMode.svg:263]: N should be lowercase

**Comment by maxwxyz** [comment]
@luzpaz ping

File: src/Gui/Icons/Std_DemoMode.svg (hunk 1 above)
Comment (luzpaz) [reply src/Gui/Icons/Std_DemoMode.svg]: Fixed, thanks for catching that.

**Comment by luzpaz** [comment]
@maxwxyz Done

**Pull request merged**